src/controllers/currentWarningsController.js

```diff
@@ -0,0 +1,140 @@
+/* eslint-disable */
+const mongoose = require('mongoose');
+const userProfile = require('../models/userProfile');
+
+const currentWarningsController = function (currentWarnings) {
+  const checkIfSpecialCharacter = (warning) => {
+    return !/^\b[a-zA-Z]+\b.*$/.test(warning);
+  };
+  const getCurrentWarnings = async (req, res) => {
+    try {
+      const response = await currentWarnings.find({});
+
+      if (response.length === 0) {
+        return res.status(400).send({ message: 'no valid records' });
+      }
+      return res.status(201).send({ currentWarningDescriptions: response });
```

Comment:
Great Implementation however, I suggest you use the right status code for a GET request as `201` is used for the successful creation of a resource and should be `200` instead.